kube/kd/deployment.yaml

```diff
@@ -43,24 +43,12 @@ spec:
                   name: {{$.KUBE_NAMESPACE}}-{{lower .}}-rds
                   key: host
                   optional: true
-            - name: {{upper .}}_DB_PORT
```

Comment:
If the port's in every secret we may as well keep this, just in case something changes in future.